src/Math-Tests-Complex/PMComplexTest.class.st

```diff
@@ -178,6 +178,36 @@ PMComplexTest >> testBug1 [
 	self assert: (0.5 * (2 + 0 i) ln) exp equals: (0.5 * 2 ln) exp
 ]
 
+{ #category : #tests }
+PMComplexTest >> testCloseTo [
+
+	self assert: 2 + 3.000000000000001i closeTo: 2 + 3i.
+	self assert: 2.000000000000001 + 3i closeTo: 2 + 3i.
+	self assert: 2.000000000000001 + 3.000000000000001i closeTo: 2 + 3i.
+]
+
+{ #category : #tests }
+PMComplexTest >> testCloseToReal [
```

Comment:
What about a test where you compare a complex number with a real?

E.g. deny: 2 + 3 i closeTo: 2.000001

Reply:
Done